packages/hardhat-core/src/builtin-tasks/compile.ts

```diff
@@ -157,6 +158,18 @@ subtask(TASK_COMPILE_SOLIDITY_READ_FILE)
     }
   );
 
+/**
+ * This task is meant to be overriden by potential plugins
+ * It allows to transform an import name
```

Comment:
```suggestion
 * This task transform the string literal in and import directive.
 * By default it does nothing, but it can be overriden by plugins.
```

Reply:
Done in 9ada664159680661a76570bc5a720d41b4580d85 because the suggestion has a typo.